### PR TITLE
Privacy + copyright emails + copyright survey

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

File: en-US/emails/2017/copyright.lang [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# Do not translate [name]
+;Hello [name],
+Hello [name],
+
+
+;Last year, over 100,000 people signed a petition for copyright reform — and stood up for innovation and user generated content like memes. Most importantly, you made it clear that some of the best parts of the Internet — from cat memes to educational resources — should be safeguarded and <b>not</b> made harder to access.
+Last year, over 100,000 people signed a petition for copyright reform — and stood up for innovation and user generated content like memes. Most importantly, you made it clear that some of the best parts of the Internet — from cat memes to educational resources — should be safeguarded and <b>not</b> made harder to access.
+
+
+;Now it’s 2017 and things are really starting to happen in the European Parliament on copyright reform. Updated drafts of the legislation are being written and these next few months will be crucial in shaping the finali revised copyright law.
```
Comment (TheoChevalier) [inline en-US/emails/2017/copyright.lang]: Waiting for clarification about finali

File: en-US/emails/2017/copyright.lang [hunk 2]
```diff
@@ -0,0 +1,81 @@
+## NOTE: Email to remind people we’re doing the campaign and ask them to share their story
+
+
+# Subject candidate 1
+;Share your story —
+Share your story —
```
Comment (flodolo) [inline en-US/emails/2017/copyright.lang]: Why the -? What's after that?

File: en-US/emails/2017/copyright.lang [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+;Will you share your story about why copyright reform matters to you?
+Will you share your story about why copyright reform matters to you?
+
+
+;Your story doesn’t have to be long or complex - even a simple statement about how you use the web will help decision-makers better understand this issue.  For example, Astrid from Berlin shares, “I hate seeing political mashups be taken down because of copyright - that commentary shouldn’t be silenced”  and Theo from Paris believes, “These laws were written for a time before cell phones. We need laws that will help businesses innovate not make their lives harder”.
+Your story doesn’t have to be long or complex - even a simple statement about how you use the web will help decision-makers better understand this issue.  For example, Astrid from Berlin shares, “I hate seeing political mashups be taken down because of copyright - that commentary shouldn’t be silenced”  and Theo from Paris believes, “These laws were written for a time before cell phones. We need laws that will help businesses innovate not make their lives harder”.
```
Comment (flodolo) [inline en-US/emails/2017/copyright.lang]: American style double space after punctuation.

Is it you being quoted here?

File: en-US/emails/2017/copyright.lang (hunk 2 above)
Comment (TheoChevalier) [inline en-US/emails/2017/copyright.lang]: The preheader

File: en-US/emails/2017/copyright.lang (hunk 3 above)
Comment (TheoChevalier) [inline en-US/emails/2017/copyright.lang]: Heh, not me :)

File: en-US/emails/2017/copyright.lang (hunk 2 above)
Comment (flodolo) [inline en-US/emails/2017/copyright.lang]: So, it's either "Subject 1 + preheader" or "Subject2"? I would add it to the note anyway

Comment (TheoChevalier) [inline en-US/emails/2017/copyright.lang]: Maybe I should duplicate the preheader and have one starting with — and one without, but that’s a bit overkill

Comment (flodolo) [inline en-US/emails/2017/copyright.lang]: It might be clearer, and safer if you want to change it a bit

**Pull request merged**